lib/qless/middleware/retry_exceptions.rb

```diff
@@ -48,19 +48,16 @@ def on_retry_callback
         @on_retry_callback ||= DEFAULT_ON_RETRY_CALLBACK
       end
 
-      def exponential(base, options = {})
+      # If`factor` is omitted it is set to `delay_s` to reproduce legacy
```

Comment:
Would be nice to have a space between "If" and "`factor`.

Also, s/delay_s/delay_seconds/ since that's what the arg is called now.